tapo/src/api/hub_handler.rs

```diff
@@ -58,6 +58,37 @@ impl HubHandler {
         self.client.read().await.get_device_info().await
     }
 
+    /// Returns a list of alarm types (ringtones) supported by the hub.
+    pub async fn get_supported_alarm_type_list(&self) -> Result<Vec<String>, Error> {
```

Comment:
I started to using ringtone based on our conversations, I don't feel strongly either way. Ringtone is what they call it in the app. But "alarm type" is what their API calls it. I've been over (I think) all uses and fixed it such that ringtone is used on any public methods from this crate, but I've left the "alarm_type" name internally since that is what their API calls it. Are you happy with that split?